snafu/hammerdb/trigger_hammerdb.py

```diff
@@ -94,9 +94,10 @@ def _parse_stdout(self, stdout):
         data = []
         for line in stdout.splitlines():
             if "TEST RESULT" in line:
-                worker = (line.split(":"))[0]
-                tpm = (line.split(" "))[6]
-                nopm = (line.split(" "))[-2]
+                worker_name = (line.split())[1]
+                worker = int((worker_name.split(":"))[0])
+                tpm = int((line.split())[-3])
```

Comment:
got this error while testing it locally

`Traceback (most recent call last):
  File "/usr/local/bin/run_snafu", line 33, in <module>
    sys.exit(load_entry_point('snafu', 'console_scripts', 'run_snafu')())
  File "/opt/snafu/snafu/run_snafu.py", line 118, in main
    for i in process_generator(index_args, parser):
  File "/opt/snafu/snafu/run_snafu.py", line 138, in process_generator
    for action, index in data_object.emit_actions():
  File "/opt/snafu/snafu/hammerdb/trigger_hammerdb.py", line 236, in emit_actions
    data = self._parse_stdout(stdout[0])
  File "/opt/snafu/snafu/hammerdb/trigger_hammerdb.py", line 99, in _parse_stdout
    tpm = int((line.split())[-3])
ValueError: invalid literal for int() with base 10: 'SQL'`

Reply:
(worker_name.split(":"))[0]  return 'SQL' I guess

Reply:
`(line.split())[-3] ` return 'SQL'

Reply:
Thx Amit, Fix it in the last commit. (parse according to database type)